Sources/Stevenson/SlackService.swift

```diff
@@ -78,10 +80,23 @@ public struct SlackService {
             return try SlackResponse(command.help)
                 .encode(for: request)
         } else {
-            return try command
-                .run(metadata, request)
-                .mapIfError { SlackResponse($0.localizedDescription, visibility: .user) }
+            return try SlackResponse("Ok!")
                 .encode(for: request)
+                .always {
+                    _ = try? command
+                        .run(metadata, request)
+                        .mapIfError {
+                            SlackResponse($0.localizedDescription, visibility: .user)
+                        }
+                        .thenThrowing { response in
+                            try request.client()
+                                .post(metadata.responseURL, headers: ["Content-type": "application/json"]) {
+                                    try $0.content.encode(response)
+                                }
+                                .catchError(.capture())
+                                .hopTo(eventLoop: request.eventLoop)
```

Comment:
I don't completely understand how it works, but this should keep the original request event loop alive so that our `client.post` actually is performed

Reply:
Any documentation or look on what suggested you to write it that way? Was it inspired from similar code you saw for another Vapor project or in their forums? (I'm asking because I'm on my phone so hard to browse and find the Vapor doc as easily)

The `thenThrowing` and `hopTo` are indeed not natural to read and would need some doc comments. Well could also reduce the pyramid of doom here by using either a nested or a private function.

Reply:
sorry, there is no doc for my "hunch" 😄 
this seems to be not needed in the end and works without it